packages/genesys-spark-tokens/snapshot/json/gse-ui-dataDisplay.json

```diff
@@ -386,7 +390,15 @@
         }
       }
     },
-    "focus": "#7B88F7"
+    "focus": "#7B88F7",
+    "add-change-image": {
+      "hover-modal": {
+        "shroud-size": "84px"
+      },
+      "icon": {
+        "size": "16px"
```

Comment:
There is an icon.size.sm or something like that.

Reply:
I think it's simpler to remove these sizes and use the icon prop for size. I'll check with Monse about updating this